hw/ip/i2c/README.md

```diff
@@ -41,21 +41,21 @@ These protocols are not in scope of this document.
 
 ## Description
 
-This IP block implements the [I2C specification](https://www.nxp.com/docs/en/user-guide/UM10204.pdf), though with some variation in nomenclature.
-For the purposes of this document, a "I2C Host" meets the specifications put forth for a "Master" device.
-Furthermore, a device which meets the specifications put forward for an I2C "Slave" device is here referred to as an "I2C Target" or "I2C Target Device".
+This IP block implements the [I2C specification](https://web.archive.org/web/20210813122132/https://www.nxp.com/docs/en/user-guide/UM10204.pdf), though with some variation in nomenclature.
+For the purposes of this document, an "I2C Host" meets the specifications put forth for a "Master" device.
+Furthermore, a device which meets the specifications put forward for an "I2C Slave" device is here referred to as an "I2C Target" or "I2C Target Device".
 
 At a high-level, the I2C protocol is a clock-parallel serial protocol, with at least one host issuing transactions to a number of targets on the same bus.
 Though I2C optionally allows for multiple hosts on the same bus, we do not support this feature at this time.
 
 Every transaction consists of a number of bytes transmitted, either from host-to-target or target-to-host.
 Each byte is typically followed by a single bit acknowledgement (ACK) from the receiving side.
 Typically each transaction consists of:
-1. A START signal, issued by host.
-1. An address, issued by host, encoded as 7 bits.
-1. A R/W bit indicating whether the transaction is a read from the target device, or a write to the target device.
+1. A START signal, issued by the host.
```

Comment:
Should these items be bullets, since they are all numbered `1.`? Or is the ordering import here, in which case the numbers should increment.